playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -566,8 +570,9 @@ private MediaItem buildInternalExoMediaItem(PKMediaSourceConfig sourceConfig, Li
         PKMediaFormat format = sourceConfig.mediaSource.getMediaFormat();
         PKRequestParams requestParams = sourceConfig.getRequestParams();
 
-        if (format == null) {
-            throw new IllegalArgumentException("Unknown media format: " + format + " for url: " + requestParams.url);
+        if (format == null || TextUtils.isEmpty(requestParams.url.toString())) {
+            return null;
+           // throw new IllegalArgumentException("Unknown media format: " + format + " for url: " + requestParams.url); // workaround for FEC-11189
```

Comment:
can be removed and add comment about the change

Reply:
Done